hang_man/main.js

```diff
@@ -0,0 +1,114 @@
+const mystery_words_array = [
+  "Malamute",
+  "Poodle",
+  "Labrador",
+  "Spaniel",
+  "Bloodhound",
+  "Sheppard",
+  "Weiner",
+  "Bulldog",
+  "Boxer",
+  "Chihuahua",
+  "Mutt",
+  "Weimaraner",
+  "Pug"
+]
+const mystery_word = mystery_words_array[Math.floor(Math.random()*mystery_words_array.length)].toLowerCase();
+console.log(mystery_word)
+const mystery_letter_array = mystery_word.split("");
+let wrong_attempt_counter = 0;
+let win = 0;
+const you_win_sound = new Audio(`sounds/win.wav`);
+const you_loose_sound = new Audio(`sounds/fart.wav`);
+
+$(document).ready(function (){
+
+  drawKeyboard()
+  drawMysteryWord(mystery_word)
+
+  document.addEventListener('keydown', event => {
+    const { keyCode } = event;
+    guessLetter(keyCode)
+  })
+  document.querySelectorAll('.letter').forEach(node => {
+    node.addEventListener('click', event => {
+      let keyCode = event.currentTarget.id
+      guessLetter(keyCode)
+    }) 
+
+  })
+  document.querySelector("#hint-button").addEventListener("mouseenter", () => {
+    document.querySelector("#hint-button").classList.add("d-none");
+    document.querySelector("#cheater").classList.remove("d-none");
+    document.querySelector("#hint").classList.remove("d-none");
+  });
+  document.querySelector("#cheater").addEventListener("mouseleave", () => {
+    document.querySelector("#hint-button").classList.remove("d-none");
+    document.querySelector("#cheater").classList.add("d-none");
+    document.querySelector("#hint").classList.add("d-none");
+  });
+})
+drawKeyboard = () => {
+  for (let charCode = 65; charCode < 91; charCode++) {
+    const div = document.createElement("div");
+    div.classList.add("col-auto", "p-1");
+    const button = document.createElement("button");
+    button.setAttribute("id", charCode)
+    button.classList.add("btn", "btn-outline-info", "btn-lg", "letter")
+    const letter = document.createTextNode(String.fromCharCode(charCode));
+    button.appendChild(letter);
+    div.appendChild(button);
+    document.getElementById("letters").appendChild(div); 
```

Comment:
Nitpick: You don't need to fin the `#letters` element every time. You can store this in a variable outside of the `for` and then just append to it
```javascript
const letters = document.getElementById('letters');
for(...) {
  // ...
  letters.appendChild(div)
}
```